sklearn/metrics/classification.py

```diff
@@ -822,6 +822,21 @@ def precision_recall_fscore_support(y_true, y_pred, beta=1.0, labels=None,
     >>> precision_recall_fscore_support(y_true, y_pred, average='weighted')
     ... # doctest: +ELLIPSIS
     (0.22..., 0.33..., 0.26..., None)
+    >>> precision_recall_fscore_support(y_true, y_pred, average=None)
+    ... # doctest: +ELLIPSIS,+NORMALIZE_WHITESPACE
+    (array([ 0.66...,  0. ,  0. ]),
+     array([ 1.,  0.,  0.]),
+     array([ 0.8,  0. ,  0. ]),
+     array([2, 2, 2]))
```

Comment:
I would drop this example as the ordering of the result is implicit in that case.